narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,6369 @@
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Early spread within Asia](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-01-15&p=full)
+
+Here, we highlight samples from mid-December 2019 through mid-January 2020, during the first month of the outbreak.
+Tips of the tree represent samples from infected individuals.
+Each internal node, or branch point, represents an unsampled case, which was the ancestor of all of the other cases within the descendant clade.
+<br><br>
+Tips are colored according to where they were sampled from; nodes (branch points) are colored by the inferred (most likely) location of the ancestral case.
+<br><br>
+Comparing where a sequence was isolated from geographically to where its ancestors were circulating can help us understand not only where infections are occurring, but also where they may have been acquired.
+<br><br>
+The common ancestor of circulating viruses appears to have emerged in Wuhan, China, in late Nov or early Dec 2019.
+Accordingly, the majority of sampled and inferred ancestral cases were located in Asia during this early period.
+This is consistent with circulation largely within China and some exported cases to other countries in Asia for the first month of the outbreak.  
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Initial introductions to other regions](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-02-15&dmin=2020-01-15&p=full&d=map,tree)
+
+During the 2nd month of the outbreak, from mid-January to mid_February 2020, we start to see individual cases crop up in North America, Europe and Oceania.
+These early samples still group with samples from Asia, and generally lack the hallmarks of local transmission.
+<br><br>
+This is consistent with initial sparks being thrown from the main outbreak in Asia to other parts of the world. Although most of these sparks did not result in localized outbreaks, at least two of these introductions took off.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Cryptic transmission in Europe from late January](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmax=2020-02-14&label=clade:A2&p=full)
+
+Zooming in to the top of the tree for the same time period, we see an ancestral European virus (internal node) that seeded the large European outbreak.
+<br><br>
+This indicates that while we hadn't yet sequenced any samples from this clade, the origins of the European outbreak were already circulating undetected by late January.
```

Comment:
I'd agree with Nicola here - "undetected by late January to early February." I've got a lot of journalists asking questions about this type of stuff at the moment and I'd rather us err on the side of caution and open intervals here rather than spend ages trying to beat them back later....